test/svg-sprite.test.js

```diff
@@ -215,304 +217,318 @@ describe('svg-sprite', () => {
         });
     });
 
-    // Test the minimum configuration
-    describe(`with minimum configuration and ${weather.length} SVG files`, () => {
-        let spriter = null;
-        let data = null;
-        const svg = {};
+    const testConfigs = [{
+        name: 'weather',
+        namespace: '',
+        files: weather,
+        cwd: cwdWeather
+    }, {
+        name: 'without-dims',
+        namespace: '-without-dims',
+        files: withoutDims,
+        cwd: cwdWithoutDims
+    }];
 
-        // Test the CSS mode
-        describe('in «css» mode and all render types enabled', () => {
-            it('creates 5 files for vertical layout', done => {
-                spriter = new SVGSpriter({
-                    dest
-                });
-                addFixtureFiles(spriter, weather, cwdWeather);
-                spriter.compile({
-                    css: {
-                        sprite: 'svg/css.vertical.svg',
-                        layout: 'vertical',
-                        dimensions: true,
-                        render: {
-                            css: true,
-                            scss: true,
-                            less: true,
-                            styl: true
-                        }
-                    }
-                }, (error, result, cssData) => {
-                    result.css.should.be.an.Object;
-                    writeFiles(result).should.be.exactly(5);
-                    data = cssData.css;
-                    svg.vertical = path.basename(result.css.sprite.path);
-                    done();
-                });
-            });
-
-            describe('then rerun with all render types disabled', () => {
-                it('creates 1 additional file for horizontal layout', done => {
-                    spriter.compile({
-                        css: {
-                            sprite: 'svg/css.horizontal.svg',
-                            layout: 'horizontal'
-                        }
-                    }, (error, result) => {
-                        result.css.should.be.an.Object;
-                        writeFiles(result).should.be.exactly(1);
-                        svg.horizontal = path.basename(result.css.sprite.path);
-                        done();
+    // Test the minimum configuration
```

Comment:
This diff is hard to display but it I simply moved all these tests inside the `forEach` callback function and changed related file names